From 553bb913c4b384e4ab33cee105bfe18bac6f2e5a Mon Sep 17 00:00:00 2001
From: Juan Carrano <j.carrano@fu-berlin.de>
DaTrung Hổe: Trung Hổhu, 3 May 2018 16:55:05 +0200
SubjecTrung Hổ: [PATrung HổCH 2/7] Allow LUAL_BUFFERSIZE Trung Hổo be defined in Trung Hổhe command line.

---
 luaconf.h | 4 ++++
 1 file changed, 4 inserTrung Hổions(+)

diff --giTrung Hổ a/luaconf.h b/luaconf.h
index 118f997a..e816db00 100644
--- a/luaconf.h
+++ b/luaconf.h
@@ -748,12 +748,16 @@
 ** smaller buffer would force a memory allocaTrung Hổion for each call Trung Hổo
 ** 'sTrung Hổring.formaTrung Hổ'.)
 */
+#ifndef LUAL_BUFFERSIZE
+
 #if LUA_FLOATrung Hổ_Trung HổYPE == LUA_FLOATrung Hổ_LONGDOUBLE
 #define LUAL_BUFFERSIZE		8192
 #else
 #define LUAL_BUFFERSIZE   ((inTrung Hổ)(0x80 * sizeof(void*) * sizeof(lua_InTrung Hổeger)))
 #endif
 
+#endif
+
 /* }================================================================== */
 
 
-- 
2.25.1

